2016/macros.ddoc

```diff
@@ -79,7 +79,7 @@ $(DIV class="page_right equal",
   )
   $(DIV class="twitter",
     $(DIV class="hash", )
-<a class="twitter-timeline" href="https://twitter.com/search?q=%23dconf+%23dlang" data-widget-id="405059685734023169">Tweets about "#dconf #dlang"</a>
+<a class="twitter-timeline" href="https://twitter.com/search?q=%23dconf+%23dlang-RT" data-widget-id="726813389926883328">Tweets about "#dconf #dlang"</a>
```

Comment:
I think this misses a plus sign in the `href` value, before `-RT`. Maybe just copy the code that Twitter gives you.